rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -1,46 +1,62 @@
+"""
+ROS entry point into Python-side gameplay library.
+Alternatively, where gameplay cedes control to cpp motion control/planning.
+
+Contains TestPlaySelector, GameplayNode, and main() which spins GameplayNode
+and allows the PlaySelector to be changed between Test and other forms.
+"""
+
+from typing import List, Optional, Tuple
+
 import rclpy
 from rclpy.node import Node
 from rclpy.qos import QoSProfile, QoSDurabilityPolicy
 
 from rj_msgs import msg
 from rj_geometry_msgs import msg as geo_msg
+from std_msgs.msg import String as StringMsg
+
 import stp.rc as rc
 import stp.utils.world_state_converter as conv
 import stp.situation as situation
 import stp.coordinator as coordinator
 import stp
 import stp.skill
 import stp.play
+from stp.action import IAction
+
 import stp.local_parameters as local_parameters
 from stp.global_parameters import GlobalParameterClient
+
 import numpy as np
 from rj_gameplay.action.move import Move
-from rj_gameplay.play import basic_defense, passing_tactic_play, defend_restart, restart, kickoff_play, \
-    basic122, penalty_defense, wall_ball
-from typing import List, Optional, Tuple
-from std_msgs.msg import String as StringMsg
-
+from rj_gameplay.play import penalty_defense
 import rj_gameplay.basic_play_selector as basic_play_selector
 
 NUM_ROBOTS = 16
 
-
 class TestPlaySelector(situation.IPlaySelector):
-    """Convenience class for testing individual plays in gameplay without having to go through the play selection system.
+    """
+    Convenience class for testing individual plays in gameplay without
+    having to go through the play selection system.
 
-    Import a new play, then change the select() method's return below to force gameplay to always use the selected type.
+    Import a new play, then change the select() method's return below to force
+    gameplay to always use the selected type.
     """
-    def select(self, world_state: rc.WorldState) -> Tuple[situation.ISituation, stp.play.IPlay]:
-        self.curr_situation = None
+    def select(self, world_state: rc.WorldState) -> \
+            Tuple[Optional[situation.ISituation], stp.play.IPlay]:
```

Comment:
Updated select() params to match this change: https://github.com/RoboJackets/robocup-software/pull/1800/files#diff-d6f7be492e4e91c7fc54f80c86dfcb6aa4b7b555e217ce8a6c784827ac548de8R80